stdlib/public/core/StringLegacy.swift

```diff
@@ -238,7 +231,7 @@ extension String {
 
   // FIXME: can't just use a default arg for radix below; instead we
   // need these single-arg overloads <rdar://problem/17775455>
-  
+
```

Comment:
We've decided as a project to to avoid introducing changes that strip trailing whitespace.  They add noise to the commit history and to code review, with no benefit.